packages/cfd/src/Stores/Modules/CFD/Helpers/cfd-config.ts

```diff
@@ -76,7 +76,7 @@ export const getMtCompanies = (is_eu: boolean) => {
     const all_config = {
         account_type: '',
         leverage: 100,
-        short_title: localize('Demo'),
+        short_title: localize('Swap-Free'),
```

Comment:
This configurations needs to change. it will affect all of our accounts with `all` type.
Your fix is temporary and needs to be changes later.

Reply:
I agree, but these changes look like it affects many other places.

Reply:
acked on this, will look into other solutions